### PR TITLE
Expand, and better document, the default whiteslist

### LINKED ISSUE #54: Reconsider the allowed list of elements and attributes
See https://www.reddit.com/r/rust/comments/72kp3c/ammonia_100rc1_an_html_sanitization_crate/dnjdxpn/. Changing the defaults is arguably a breaking change, so we might want to do it before the stable release.

### LINKED ISSUE #55: Better document the defaults
Almost nothing says what `link_rel` is set to by default, but this applies to all configuration setters.

Courtesy of https://www.reddit.com/r/rust/comments/72kp3c/ammonia_100rc1_an_html_sanitization_crate/dnjdxpn/.

### PR DESCRIPTION
**Pull request by notriddle**
Fixes #54 

Fixes #55

### CODE REVIEW

**Comment by notriddle** [comment]
CC @ssokolow

**Comment by ssokolow** [comment]
I just posted an amendment to #54 before I saw this. Should I repost it here?

**Comment by lnicola** [comment]
@notriddle I'd rather look over this tomorrow.

**Comment by lnicola** [comment]
@ssokolow thanks for the follow-up

File: src/lib.rs [hunk 1]
```diff
@@ -65,6 +65,12 @@ lazy_static! {
 ///                `<img width="" height="" src="" alt="">`
```
Comment (lnicola) [inline src/lib.rs]: This list is now somewhat incomplete. Should we try to keep it in sync or just link to the setters?

File: src/list [hunk 2]
```diff
@@ -0,0 +1,77 @@
+a
```
Comment (lnicola) [inline src/list]: Do we want to have these here?

**Comment by lnicola** [comment]
Same for `list2`, I suppose?

**Comment by notriddle** [comment]
I made the relative-URL setting "pass-through" by default, instead of Deny. Sure, for simple comment boxes, I usually do want to block relative URLs, but the new defaults are designed for *articles*, which are going to want them.

**Pull request merged**